src/Interpolator.js

```diff
@@ -196,6 +200,7 @@ class Interpolator {
       if (match[0].includes(this.formatSeparator) && !/{.*}/.test(match[1])) {
         const r = match[1].split(this.formatSeparator).map(elem => elem.trim());
         match[1] = r.shift();
+        r.push(match[1]);
```

Comment:
Can you add a specific test that uses this code part?

Reply:
Removed line.  It wasn't necessary and did case an issue.